northstar/src/runtime/process/init.rs

```diff
@@ -30,11 +30,130 @@ pub(super) struct Init {
     pub seccomp: Option<AllowList>,
 }
 
+/// This is the type used to communicate with the init process from the runtime.
+#[derive(Debug, Serialize, Deserialize)]
+pub enum InitMessage {
+    NewChild { pid: i32 },
+    DeadChild { pid: i32, exit_status: ExitStatus },
```

Comment:
Can we rename this please? I have bad memories.

```rust
pub enum Message {
   Forked? { ... },
   Exit { ...},
}

Reply:
sure :)